docs/migration_guide.rst

```diff
@@ -1,11 +1,36 @@
+0.11.0 Migration guide
+======================
+
+Cairo-lang 0.10.3 dropped support for the `Deploy` transaction. To be compatible we had to remove some
+(already deprecated) features.
+
+Breaking Changes
+----------------
+
+Removed methods:
+
+- `Contract.deploy`. Read more about deployment in the `Deploying contracts <https://starknetpy.readthedocs.io/en/latest/guide.html#deploying-contracts>`_ section.
+- `AccountClient.create_account`. `Account creation <https://starknetpy.readthedocs.io/en/latest/account_creation.html>`_ docs are here to help you!
+- `Client.deploy` method (from the interface and all implementations)
+- `make_deploy_tx`
+- `compute_deploy_hash`
+- the `Deploy` transaction
+
+
+Invoke Transaction
+------------------
+
+Old `InvokeFunction` transaction is now aliased as `Invoke`. We suggest to start using the new `Invoke`.
```

Comment:
I suggest moving it above breaking changes.

Reply:
See your point, but `Breaking changes` are always on the top

Reply:
I'd keep breaking changes where they are, it's the most important thing from the user perspective when they check migration guide.